src/main/java/ro/hiringsystem/service/AnonymousUserService.java

```diff
@@ -0,0 +1,23 @@
+package ro.hiringsystem.service;
+
+import ro.hiringsystem.model.AnonymousUser;
+
+import java.util.Map;
+import java.util.Optional;
+import java.util.UUID;
+
+public interface AnonymousUserService {
+
+    Optional<AnonymousUser> getById(UUID id);
+
+    Map<UUID, AnonymousUser> getAllFromMap();
+
+    void addAllFromGivenMap(Map<UUID, AnonymousUser> clientMap);
+
+    void addOnlyOne(AnonymousUser anonymousUser);
```

Comment:
Syntactic sugar, the method could be renamed to 'add' instead of 'addOnlyOne' as it is clear that we only add one by it's parameter.